scripts/build/build-utils.ps1

```diff
@@ -145,18 +145,18 @@ function Invoke-UnitTests([string]$binPath, [string]$buildConfiguration) {
         }
     $testDirs = $testDirs | Select-Object -Uniq
 
-    Write-Header "Running unit tests .NET Framework 4.8"
-    & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal"  /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
-    Test-ExitCode "ERROR: Unit Tests execution FAILED."
+#     Write-Header "Running unit tests .NET Framework 4.8"
+#     & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal" /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
+#     Test-ExitCode "ERROR: Unit Tests execution FAILED."
```

Comment:
Debug artifact? Doesn't sem intentional
```suggestion
     Write-Header "Running unit tests .NET Framework 4.8"
     & (Get-VsTestPath) $testFiles /Logger:"console;verbosity=minimal" /Parallel /Enablecodecoverage /InIsolation  /TestAdapterPath:$testDirs
     Test-ExitCode "ERROR: Unit Tests execution FAILED."
```

Reply:
oh yes. I was testing and forgot to remove the comments. Thaks